### PR TITLE
Tag TaylorSeries v0.2.1

### PR DESCRIPTION
**Pull request by lbenet**

### CODE REVIEW

**Comment by dpsanders** [comment]
I wonder if this should be a minor tag, since the api has changed?

File: TaylorSeries/versions/0.2.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Compat 0.6
```
Comment (tkelman) [inline TaylorSeries/versions/0.2.1/requires]: this needs to be higher for `Compat.String`, I think 0.7.15

**Comment by lbenet** [comment]
Thanks for both advices; I'll tag a new minor version and correct the requirements.

@tkelman Should I close this PR, and then open a new PR?

**Comment by tkelman** [comment]
either a new PR, or push updates to this branch of your metadata fork

**Comment by lbenet** [comment]
Closing, in favor of #6493

**Pull request closed without merging**